WalletLibrary/Submodules/VerifiableCredential-SDK-iOS/VCNetworking/VCNetworking/IssuanceNetworkCalls.swift

```diff
@@ -3,16 +3,10 @@
 *  Licensed under the MIT License. See License.txt in the project root for license information.
 *--------------------------------------------------------------------------------------------*/
 
-import PromiseKit
-
-#if canImport(VCEntities)
-    import VCEntities
-#endif
-
 protocol IssuanceNetworking {
-    func getRequest(withUrl url: String) -> Promise<SignedContract>
-    func sendResponse(usingUrl url: String, withBody body: IssuanceResponse) -> Promise<VerifiableCredential>
-    func sendCompletionResponse(usingUrl url: String, withBody nody: IssuanceCompletionResponse) -> Promise<String?>
+    func getRequest(withUrl url: String) async throws -> SignedContract
+    func sendResponse(usingUrl url: String, withBody body: IssuanceResponse) async throws -> VerifiableCredential
+    func sendCompletionResponse(usingUrl url: String, withBody body: IssuanceCompletionResponse) async throws
```

Comment:
We're changing the signature to not return?

Reply:
well this is all internal stuff that we be refactored pretty soon so it would not affect developers.